pkg/kubelet/eviction/defaults_windows.go

```diff
@@ -0,0 +1,27 @@
+//go:build windows
+// +build windows
+
+/*
+Copyright 2024 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package eviction
+
+// DefaultEvictionHard includes default options for hard eviction.
+var DefaultEvictionHard = map[string]string{
+	"memory.available":  "500Mi",
```

Comment:
On Linux, we have this at `100Mi`. Should we document the divergence here?

Reply:
I'm going to update the k8s.io website with details about how memory pressure eviction will work on Windows.
Would it be sufficient to mention it there?

Reply:
Sounds good!

Reply:
updated in the code too